plugins/aep-core-configuration/plugin.json

```diff
@@ -1,10 +1,10 @@
 {
-    "category": "AEP Core Configuration",
-    "displayName": "Adobe Core AEP SDK Extension Configuration",
-    "description": "Verifies the Adobe Core AEP SDK Extension has been configured.",
+    "category": "Adobe Core Configuration",
```

Comment:
What about `Adobe Core` or `AEP SDK Configuration`? I can't imagine we'll have very many validations under these categories